client-app/shared/account/composables/useOrganizationContacts.ts

```diff
@@ -0,0 +1,104 @@
+import getOrganizationContacts from "@/xapi/graphql/account/queries/getOrganizationContacts";
+import { ContactType, UserType } from "@/xapi/types";
+import { ref, shallowRef, Ref, readonly, computed } from "vue";
+import { Logger } from "@/core/utilities";
+import { ISortInfo } from "../types";
+import { ContactStatus, DEFAULT_PAGE_SIZE, SORT_ASCENDING } from "@/core/constants";
+import { getSortingExpression } from "../utils";
+import useUser from "./useUser";
+import _ from "lodash";
+import { OrganizationContactType } from "@/core/types";
+import { useI18n } from "vue-i18n";
+
+export default () => {
```

Comment:
maybe it be better if we pass current user to composition api fabric function instead of useUser() calling inside.